zzzz_modular_occulus/code/game/machinery/dorms_elevator.dm

```diff
@@ -158,10 +201,17 @@
 	desc = "An button used to call the elevator down to the current floor."
 	var/cooldown = 24 SECONDS
 	var/cooldown_timer
+	var/obj/machinery/computer/cryopod/elevator_storage/lostFound
 
 /obj/machinery/button/remote/elevator_call_button/trigger()
+	for(var/obj/machinery/computer/cryopod/elevator_storage/storage in range(usr, 5))
+		lostFound = storage
```

Comment:
This should be in initialize() or something, this makes it constantly update this var every time someone hits the button, which misses the whole point of making it a var to begin with.

Reply:
fixed